.github/workflows/ci.yml

```diff
@@ -305,6 +312,11 @@ jobs:
       with:
         vtk_sha_file: ./source/.github/actions/vtk_commit_sha
 
+    - name: Set LD_LIBRARY_PATH ubuntu
```

Comment:
Missing `if os == ubuntu`?

Reply:
Nothing is needed for macOS?

Reply:
Added more info to explain why it is needed.

This is a linux only and vtk < 9.2.0 only issue that requires this.

All the rest (macos and other VTK versions) rely fully on rpath to find the libs.

I have no choice because macos image have the security (SIP) enabled and prevent the use of DYLD_LIBRARY_PATH through multiple steps.

See this: 
 - https://briandfoy.github.io/macos-s-system-integrity-protection-sanitizes-your-environment/
 - https://github.com/conan-io/conan-center-index/issues/8443
 - https://github.com/pypa/cibuildwheel/issues/816
 
 An alternative would be to use DYLD_LIBRARY_PATH whenever needed, but its not as elegant and it pushes me to actually go and fix rpath issues in our deps.